src/screens/Data.js

```diff
@@ -165,6 +165,36 @@ const DataPage = () => (
   <DataTable />
 </Data>`}
     >
+      <Box>
+        <Heading level={2}>Default filter types</Heading>
```

Comment:
Yeah, I was struggling about where the right place to put this is. In general, Grommet docs tends to stay pretty lean with just props -- so felt like having it on "Data" as the most parent component would be helpful, but I understand how it feels a little off.

If we had some intro sentence about Data + friends and its purpose as a component set, then followed it with default filter types, would that read better?

Reply:
That might be OK. It definitely needs a bit more that says what the Data component is and a hint of how to use it before jumping into a rather large detailed description of some deeper aspect of it. When someone first reads it, it's hard to figure out what it is before being told some deep info about filtering which honestly I hope they'd never even have to read because Data 'just does the right thing' without you knowing those details.

Reply:
Agree with it hopefully just "doing the right thing". I'll try to strike the right balance of "light info" to establish some context without going too deep.

The desire for docs on how default filters are chosen came from some feedback of folks using it.